tests/browser-tests/AnchorLinks.browser.ts

```diff
@@ -0,0 +1,27 @@
+import { BASE_URL } from '../../test-utils/baseUrl';
+import { ClientFunction, Selector, t } from 'testcafe';
+
+fixture`AnchorLinks`.page`${BASE_URL}`;
```

Comment:
The anchor link functionality is limited to the Static pages so far so I think we might want to add test cases to the [StaticPages](https://github.com/libero/reviewer-client/blob/cc398819519a070f4daa495d08dcfd2061b3a87e/tests/browser-tests/StaticPages.browser.ts) test file instead. These tests should be user story based eg: 

- as a user i would like to load in a authorguide page at a specific section
- as a user i should be linked from the coverletter guidance text directly to X section in the author guide (where should this one go? We probably want a test that takes the user through to coverletter section of form then clicking that link)